packages/@coorpacademy-app-review/sandbox/translation.mock.ts

```diff
@@ -0,0 +1,4 @@
+import {mockTranslate} from '@coorpacademy/translate';
```

Comment:
plase move this file to `src/tests/utils`. I'd rather see tests importing this mock from utils, than importing it from the sandbox enviroment.